docs/monitoring/inventory.md

```diff
@@ -81,17 +118,25 @@ The [Sematext Agent](../agents/sematext-agent) gathers the following data about
 - Total disk size
 - Disk drives information
 
-In addition, the following information about packages is gathered for Node.js, Python and DEB sources:
+Following information about packages is gathered for Node.js, Python, DEB and RPM sources:
```

Comment:
Following information about packages is gathered for Node.js, Python, DEB and RPM sources:
=>
The following information about packages is gathered for Node.js, Python, DEB and RPM sources: